nucleus/__init__.py

```diff
@@ -251,23 +254,40 @@ def list_datasets(self) -> Dict[str, Union[str, List[str]]]:
         return self.make_request({}, "dataset/", requests.get)
 
     def list_jobs(
-        self, show_completed=None, date_limit=None
```

Comment:
Add a warning that date_limit isn't working any more for good measure?

Reply:
How would we do that?

If someone now tries to call `list_jobs(date_limit=foo)` then it will give `TypeError`.
Is there a clean way to handle this?

Only way I can think of, is to introduce a temporary 2nd function.